crates/swift-bridge-ir/src/codegen/codegen_tests/transparent_enum_codegen_tests.rs

```diff
@@ -586,17 +592,20 @@ typedef struct __swift_bridge__$Option$SomeEnum { bool is_some; __swift_bridge__
     }
 }
 
-/// Verify that we generate an enum type that has a variant with one named field and one with two named fields.
-mod generates_enum_to_and_from_ffi_conversions_one_named_data_and_two_named_data {
+/// Verify that we generate an enum type that has a variant with one with two named fields and one named field.
+mod generates_enum_to_and_from_ffi_conversions_two_named_data_and_one_named_data {
     use super::*;
 
     fn bridge_module_tokens() -> TokenStream {
         quote! {
             #[swift_bridge::bridge]
             mod ffi {
+                extern "Rust" {
+                    type SomeType;
+                }
                 enum SomeEnum {
                     A{
-                        data1: i32,
+                        data1: SomeType,
```

Comment:
Instead of modifying these tests we can add a new test that is dedicated to opaque Rust data in enums

```rust
mod enum_with_opaque_rust_data {
// ...

#[swift_bridge::bridge]
mod ffi {
    extern "Rust" {
        type SomeType;
    }

    enum SomeEnum {
        Unnamed(SomeType),
        Named { data: SomeType }
    }
}

// ...
}
```

---

Otherwise, just like how you just replace `i32` with `SomeType`, someone might come later and replace `SomeType` with `String`, not realizing that we put `SomeType` here intentionally.